### PR TITLE
Abhishek.kumar1/rsmb 51 support for geography data type

### PR DESCRIPTION
**Pull request by Abhi-210793**

### CODE REVIEW

File: core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java [hunk 1]
```diff
@@ -2468,6 +2468,8 @@ private boolean isIntervalHourAndSecond(SqlCall call) {
       case JSON:
       case VARIANT:
         return createSqlDataTypeSpecByName("JSON", typeName);
+      case GEOGRAPHY:
+        return createSqlDataTypeSpecByName("GEOGRAPHY", typeName);
```
Comment (rushikeshsapate-dm) [inline core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java:2472]: add corresponding TC

Comment (Abhi-210793) [reply core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java:2472]: done

**Review by sauajmera — APPROVED** [review]
changes looks good.

**Review by anuragsaxena2k7 — APPROVED** [review]
approved

**Pull request merged**